### PR TITLE
Add FAQ: Can I deploy when translations are missing

### PR DESCRIPTION
**Pull request by moritztk**

### CODE REVIEW

**Comment by moritztk** [comment]
Pending until we actually support a second language

**Comment by moritztk** [comment]
We actually already deployed german, it is important.

File: desk_reference/localization.md [hunk 1]
```diff
@@ -131,6 +133,10 @@ Multiple options:
 1.  [VSCode extension](https://github.com/matthewferreira/uuid-gen-extension)
 1.  [DuckDuckGo](https://duckduckgo.com/?q=uuid&t=h_&ia=answer)
 
+### Can I deploy if translations are missing?
+
+No, unless you ship a feature to english speakers only. Check with your PO.
```
Comment (maldo) [inline desk_reference/localization.md]: I will add here a How do you ship a feature to english speakers only?

Comment (moritztk) [reply desk_reference/localization.md]: I don't think there is a way to properly do that right now.

**Pull request merged**